src/App.vue

```diff
@@ -65,13 +65,7 @@ function dismissLoader() {
   }
 }
 
-onMounted(async () => {
-  loader.value = await loadingController
-    .create({
-      message: translate("Click the backdrop to dismiss."),
-      translucent: true,
-      backdropDismiss: true
-    });
+onBeforeMount(async () => {
```

Comment:
Just move the emitters in the beforeMount hook, and keep the userProfile `if` block in mounted hook only.